plugins/omar-wms-plugin/grails-app/controllers/omar/wms/WmsController.groovy

```diff
@@ -148,7 +148,7 @@ where:
 			@ApiImplicitParam( name = 'bbox', value = 'Bounding box', defaultValue = "-180,-90,180,90", paramType = 'query', dataType = 'string', required = true ),
 			@ApiImplicitParam( name = 'width', value = 'Width of result image', defaultValue = "1024", paramType = 'query', dataType = 'integer', required = true ),
 			@ApiImplicitParam( name = 'height', value = 'Height of result image', defaultValue = "512", paramType = 'query', dataType = 'integer', required = true ),
-			@ApiImplicitParam( name = 'format', value = 'MIME Type of result image', defaultValue = "image/jpeg", allowableValues = "image/jpeg, image/png, image/gif", paramType = 'query', dataType = 'string', required = true ),
+			@ApiImplicitParam( name = 'format', value = 'MIME Type of result image', defaultValue = "image/jpeg", allowableValues = "image/jpeg, image/png, image/gif, image/jpeg-png", paramType = 'query', dataType = 'string', required = true ),
```

Comment:
I don't believe adding a fake MIME type is the way to go. Similar functionality could be performed by making the format optional, and defaulting the the optimal format